actors/builtin/reward/reward_actor.go

```diff
@@ -142,15 +139,20 @@ func (a Actor) UpdateNetworkKPI(rt vmr.Runtime, currRealizedPower *abi.StoragePo
 
 	var st State
 	rt.State().Transaction(&st, func() interface{} {
-		newBaselinePower := a.newBaselinePower(&st)
-		st.BaselinePower = newBaselinePower
-		st.CumsumBaseline = big.Add(st.CumsumBaseline, st.BaselinePower)
-
 		st.RealizedPower = *currRealizedPower
 		st.CumsumRealized = big.Add(st.CumsumRealized, *currRealizedPower)
```

Comment:
I think this needs to be added as many times as there are iterations in the loop below.

Reply:
what @anorth is saying is correct but not exactly right because of the edge cases of null rounds. If there is a null round and UpdateNetworkKPI is called in the round after that, we need to first add the RealizedPower at the null round (before the update) to CumsumRealized to account for the null round, and then update RealizedPower to currRealizedPower and then add that to CumsumRealized to account for the change in power in this round.